pkg/executor/config/config.go

```diff
@@ -17,6 +17,7 @@ package config
 
 // ExecutorConfig represents the configuration for the executor
 type ExecutorConfig struct {
-	RequestTimeout *int `json:"requestTimeout"`
+	VerificationRequestTimeout *int `json:"verificationRequestTimeout"`
```

Comment:
I personally think it should be `mutation` since we now have 2 distinct timeouts for the /mutate and /verify handlers. The mutation handler is only responsible for tag to digest mutation and has no verification or validation logic. I am open to other thoughts though.